src/gui_impl/dialogs/event_types_selection_dialog.cpp

```diff
@@ -48,8 +48,8 @@ void EventTypesSelectionDialog::buildTree (bool only_existing_events)
     ui_.tree_widget_->setColumnWidth(ID_COLUMN_INDEX_, 0);
 
     ui_.tree_widget_->header()->setSectionResizeMode (QHeaderView::Interactive);
-    ui_.tree_widget_->header()->resizeSection (NAME_COLUMN_INDEX_, 380);
-    ui_.tree_widget_->header()->resizeSection (COLOR_COLUMN_INDEX_, 120);
+    ui_.tree_widget_->header()->resizeSection (NAME_COLUMN_INDEX_, width() / 1.65);
+    ui_.tree_widget_->header()->resizeSection (COLOR_COLUMN_INDEX_, width() / 5.5);
```

Comment:
Hmm looks like this way this is too much space in the middle, unless we want to shrink overall size. However, I thought bigger dialogs look nicer, also it's a trend machine's resolution is getting higher and higher

![image](https://cloud.githubusercontent.com/assets/20526646/23056646/658d198e-f4ba-11e6-8cee-f1e32cd2aeaa.png)